CI/scripts/test.sh

```diff
@@ -473,6 +488,25 @@ function test-run_example-simple_app() {
     popd > /dev/null 2>&1
 }
 
+# #############################################################################
+function test-simple_app-with-crypto_algorithms() {
+    echo "******************************************************************"
+    echo "* Test: Execute script to compile, build and run simple_app."
+    echo "******************************************************************"
+    echo " "
+    pushd ./sample_apps > /dev/null 2>&1
+
+    ./cleanup.sh
+
+    if [ "${Simple_app_setup_done}" = 0 ]; then
+        ./run_example.sh simple_app setup
+    fi
+    ./run_example.sh simple_app run_test-crypto_algorithms
```

Comment:
Just a driver method in `test.sh` to call into the testing method in the sub-script.